### ISSUE TITLE
DCTL Bug in Davinci 16 and 17 versions. HELP!

### ISSUE THREAD

**Issue by IsaacRivasCayuela** [roukou3/DCTL#3]
Sorry if I've been repeating my self but Youtube comments doesn't seem to work and I don't know how to reach to you.

There's a problem with all DCTL in Davinci 16 and 17 versions with 4:3 ratio, after converting it stays in 4:3 crop (screenshot) https://drive.google.com/file/d/1hDVEBXysD7zaTIykE05jAjLpGTixygLh/view?usp=sharing
With Davinci 15 works good, I'm using MacOS (OpenCL enabled), do you have the same problem? Can I do something to solve this?
I'd really appreciate to make it work, my Davinci 15 destroys any audio quality and makes impossible audio mixing for me, also Davinci 17 has more color tools and more advanced, including better noise reduction results.
Right now I'm exporting Prores from 15 to 17 version but my poor quality of the compressed 8-bit videos is being lost in the way.
This is a recent video example I've made of your Ptgui DCTL rendered with Davinci 15 (audio was created entirely separate with Logic): https://youtu.be/WuJv0gcKcQs
Looks awesome! Let me thank you with a LUT collection I've made public (completely legal): https://drive.google.com/drive/folders/1q_GLXKsYYDUckpdd10VzGsl67mWoBj2T?usp=sharing
Thank you

**Comment by roukou3** [comment]
I just confirmed the same problem as yours with the latest DavinciResolve.
(DaVinci Resolve Studio 17.3.1 Windows)

Apparently, DCTL is now specified to be applied to the clip resolution, not the timeline resolution.

A simple solution is to change the aspect ratio of the clip and DCTL.

Since the clip resolution is 4:3 and the timeline resolution is 16:9, multiplying horizontally by 1.333 (=16/12) will fill the timeline.

1. Specify the Zoom X of the clip placed in the timeline to be 1.333.
2. Then set DCTL's adjust:aspect to 1.333.


Your video looks great :)

**Comment by roukou3** [comment]
![capture](https://user-images.githubusercontent.com/46643575/135859822-1d936f23-dbea-4b60-b4f0-2d4bc7e81e03.jpg)

**Comment by IsaacRivasCayuela** [comment]
Thanks! It works perfectly!
Didn't think about that, what I was doing is making a "compound clip" with a solid background to trick Resolve to think it's a 16:9 clip, it worked but turns out Resolve doesn't allow Noise reduction in "compound clips" so a nightmare.

Forgot to say the LUTs I linked before are intended for rec.709 gamma 2.2 in theory.

By the way, yesterday I found an open source project for extracting gyro data (from cameras like GoPro etc) to stabilise video even from any other camera like a mirrorless with a fisheye, the trick would be to attach the two cameras together (a gyro cam and a cinema cam): 
-Video demonstration https://www.youtube.com/watch?v=f4YD5pGmnxM
-Github https://github.com/ElvinC/gyroflow
-Official web page https://elvinchen.com/gyroflow/
-Facebook group https://www.facebook.com/groups/fpvtools/?notif_id=1633266609294185&notif_t=group_r2j_approved&ref=notif

I thought maybe you'll find it interesting as your DCTLs have a lot in common, I tried more than once to combine your DCTL with Resolve stabilization without success.

As you saw in my video I'm combining pre-stabilized video from a wide angle of a GoPro 9 and after your Ptgui DCTL the results look spectacular, I find super useful the options of tilt as many times the camera mounted in the chest is looking down to the floor.
I'm also using your Ptgui DCTL for correcting and editing photos in Resolve as I find it more advanced than the Lightroom and Photoshop tools.

I think many people would be interested in your Ptgui DCTL tool as a universal and professional fisheye converter, with your consent I don't discard making a youtube tutorial in the future so people can learn about and reach to you; I always credit your DCTL and links in my videos.

Thank you very much again!
